src/transformers/call.js

```diff
@@ -0,0 +1,23 @@
+import { matchesPattern, mapPathString } from '../utils';
+
+
+const patterns = [
+  'require',
+  'require.resolve',
+  'System.import',
+  'jest.genMockFromModule',
+  'jest.mock',
+  'jest.unmock',
+  'jest.doMock',
+  'jest.dontMock',
+];
+
+export default function transformCall(types, nodePath, state) {
+  const calleePath = nodePath.get('callee');
+
+  if (!patterns.some(pattern => matchesPattern(types, calleePath, pattern))) {
+    return;
```

Comment:
We can avoid this return by removing the not in the condition and moving `mapPathString` inside :)